xml/System.Formats.Nrbf/SerializationRecordType.xml

```diff
@@ -13,8 +13,11 @@
     <BaseTypeName>System.Enum</BaseTypeName>
   </Base>
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Record type.</summary>
+    <remarks>
+      <para> The enumeration does not contain all values supported by the <see href="https://learn.microsoft.com/openspecs/windows_protocols/ms-nrbf/954a0657-b901-4813-9398-4ec732fe8b32"> [MS-NRBF] 2.1.2.1</see>, but only those supported by the <see cref="T:System.Formats.Nrbf.NrbfDecoder" />.
+ </para>
+    </remarks>
```

Comment:
this is not true anymore, I've sent https://github.com/dotnet/runtime/pull/105632 to address that

```suggestion
```